.github/workflows/enforce-issue-template.yml

```diff
@@ -0,0 +1,35 @@
+name: enforce issue template
+on:
+  issues:
+    types: ['opened']
+jobs:
+  enforce-issue-template:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: "Look for a matching issue template"
+      shell: bash
+      run: |
+
+        # Look for a matching issue template
+        echo "${{ github.event.issue.body }}" > actual
+        for expected in $(ls .github/ISSUE_TEMPLATE/*.md 2> /dev/null); do
+          echo -n "$(basename $expected)? "
+          if diff -rub <(grep '^#' $expected) <(grep '^#' actual) > /dev/null; then
```

Comment:
My bash-fu is insufficient here. Maybe add some comments to explain what is going on a bit? 🙂

Reply:
Process substitution! I think it reads easily enough ("diff expected actual"), and Googling for `bash <()` gets me an [answer](https://superuser.com/questions/1059781/what-exactly-is-in-bash-and-in-zsh) easily as well. I can add a comment.